CHANGELOG.md

```diff
@@ -64,6 +64,10 @@
 - Minor: Migrated /vips to Helix API. Chat command will continue to be used until February 11th 2023. (#4053)
 - Minor: Make menus and placeholders display appropriate custom key combos. (#4045)
 - Bugfix: Connection to Twitch PubSub now recovers more reliably. (#3643, #3716)
+- Minor: Migrated /uniquechat to Helix API. (#4057)
+- Minor: Migrated /uniquechatoff to Helix API. (#4057)
+- Minor: Migrated /r9kbeta to Helix API. (#4057)
+- Minor: Migrated /r9kbetaoff to Helix API. (#4057)
```

Comment:
I'm SURE we can get away with one changelog for uniquechat & r9kbeta and one for uniquechatoff & r9kbetaoff

but they have to go above `Bugfix: Connection to Twitch PubSub now recovers more reliably. (#3643, #3716)`